src/pages/dashboard.js

```diff
@@ -190,7 +190,7 @@ export const searchResults = (result) => {
             <tr>
                 ${tdTemplate}
                 <td>
-                    <button class="btn btn-outline-primary text-nowrap" data-check-in-btn-connect-id=${data.Connect_ID} data-check-in-btn-uid=${data.state.uid}>${!isCheckedIn ? `Go to Check-In` : `Go to Check-Out`}</button>
+                    <button ${!canCheckIn ? `disabled` : ``}  class="btn  ${!canCheckIn ? 'btn-secondary disabled' : 'btn-outline-primary'} text-nowrap" data-check-in-btn-connect-id=${data.Connect_ID} data-check-in-btn-uid=${data.state.uid}>${!isCheckedIn ? `Go to Check-In` : `Go to Check-Out`}</button>
```

Comment:
Nit: Might be cleaner to extract this `canCheckIn` ternary to a new const string variable (defined on 188) since it's referenced a few times.

Reply:
Done.